doc/api/documentation.md

```diff
@@ -37,34 +37,26 @@ and in the process of being redesigned.
 
 The stability indices are as follows:
 
-```txt
-Stability: 0 - Deprecated
-This feature is known to be problematic, and changes are
-planned.  Do not rely on it.  Use of the feature may cause warnings.  Backwards
-compatibility should not be expected.
-```
-
-```txt
-Stability: 1 - Experimental
-This feature is subject to change, and is gated by a command line flag.
-It may change or be removed in future versions.
-```
-
-```txt
-Stability: 2 - Stable
-The API has proven satisfactory. Compatibility with the npm ecosystem
-is a high priority, and will not be broken unless absolutely necessary.
-```
-
-```txt
-Stability: 3 - Locked
-Only fixes related to security, performance, or bug fixes will be accepted.
-Please do not suggest API changes in this area; they will be refused.
-```
+> Stability: 0 - Deprecated
+> This feature is known to be problematic, and changes are
+> planned.  Do not rely on it.  Use of the feature may cause warnings.  Backwards
+> compatibility should not be expected.
+
+> Stability: 1 - Experimental
+> This feature is subject to change, and is gated by a command line flag.
+> It may change or be removed in future versions.
+
+> Stability: 2 - Stable
+> The API has proven satisfactory. Compatibility with the npm ecosystem
+> is a high priority, and will not be broken unless absolutely necessary.
+
+> Stability: 3 - Locked
+> Only fixes related to security, performance, or bug fixes will be accepted.
+> Please do not suggest API changes in this area; they will be refused.
```

Comment:
This block doesn't look good on GitHub:
 1. This is interpreted as one blockquote, not four.
 2. Line breaks don't work.

Adding a manual line break there to fix how it looks on GitHub confuses the doctool and it treats those as separated blocks, with only the first line being colored.